### ISSUE TITLE
Bazel build FAILED on Ubuntu16.04 caused by extension file to detect MSVC for Windows

### ISSUE THREAD

**Issue by HwMohanLiu** [tensorflow/tensorflow#17917]
When using BAZEL to build TF r.17 from source on Ubuntu16.04. It failed caused by the bazel extension file located at /tensorflow/tools/def_file_filter/def_file_filter_configure.bzl, which is used to export symbols from TF DLL file for Windows. The same problem occurs for building TF-Lite also.

Commend:
> bazel build --config=opt  //tensorflow/tools/pip_package:build_pip_package
> bazel build --config=opt   //tensorflow/contrib/lite/toco:toco

Bazel output:
**ERROR:** ~/tensorflow-r1.7/tensorflow/tools/def_file_filter/def_file_filter_configure.bzl:22:1: file '@bazel_tools//tools/cpp:windows_cc_configure.bzl' does not contain symbol 'find_vc_path' (did you mean '_find_vc_path'?).
**ERROR:** ~/tensorflow-r1.7/tensorflow/tools/def_file_filter/def_file_filter_configure.bzl:23:1: file '@bazel_tools//tools/cpp:windows_cc_configure.bzl' does not contain symbol 'find_msvc_tool' (did you mean '_find_msvc_tool'?).
**ERROR:** error loading package '': Extension file 'tensorflow/tools/def_file_filter/def_file_filter_configure.bzl' has errors.

**Comment by tensorflowbutler** [comment]
Thank you for your post. We noticed you have not filled out the following field in the issue template. Could you update them if they are relevant in your case, or leave them as N/A? Thanks.
Have I written custom code
OS Platform and Distribution
TensorFlow installed from
TensorFlow version
Bazel version
CUDA/cuDNN version
GPU model and memory
Exact command to reproduce

**Comment by skye** [comment]
@yifeif are you able to comment?

**Comment by yifeif** [comment]
@HwMohanLiu which bazel version are you using?

**Comment by HwMohanLiu** [comment]
According to the problem that bazel can not fetch the sources behind a proxy, I used a modified bazel from https://github.com/ixuexi/bazel, which using wget to download sources instead. However, the problem is caused by the bazel version. I updated bazel to version to 0.11.0. The proxy problem of bazel have been solved by installing bazel without JDK, setting cntlm proxy and adding certificate to system JDK.
